CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ### Release Notes
 
+### Breaking changes
```

Comment:
Was this actually enabled with a limit by default after all? Bit us in the butt pretty hard today.